tools/urls-check

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+
+import os
+from os.path import dirname, abspath
+import sys
+import time
+import argparse
+import subprocess
+import urllib
+from urllib.request import urlopen, Request
+
+sys.path.append(os.path.join(dirname(dirname(abspath(__file__))), "bots"))
+import task
+
+sys.dont_write_bytecode = True
+
+DAYS = 3
+TASK_NAME = "Validate all URLs"
+
+def main():
+    parser = argparse.ArgumentParser(description=TASK_NAME)
+    parser.add_argument("-v", "--verbose", action="store_true", help="Verbose output")
+    opts = parser.parse_args()
+
+    task.verbose = opts.verbose
+
+    since = time.time() - (DAYS * 86400)
+
+    # When there is opened issue, don't do anything
```

Comment:
s/opened/an open/

So the idea is to close the issue on success, and the next round would reopen it instead of creating a new one? That makes sense, I just want to confirm that I understand it right.

Reply:
Not really.
1. If there is an open issue, it means this task failed the last time and therefore we don't run it again.
2. If there is no (closed) issue that has been modified in the last 7 days, we want to run this task.
2.1 If the task failed, always open a new issue (no reopening)
2.2 If the task succeeded:
2.2.1 If there is issue from the last 7*4 days, then just comment on it (it modifies it) (no reopening. We get notifications on comments, which is what we want)
2.2.2 If there is no issue from the last 7*4 days, open a new one and close it immediately

Does that make sense?